trl/trainer/sft_trainer.py

```diff
@@ -932,6 +932,7 @@ def _func(example):
                     column_names = next(iter(dataset)).keys()
                     dataset = dataset.map(
                         maybe_convert_to_chatml,
+                        batched=True,
```

Comment:
same here, I don't think using batched here speeds things. Unless it does (to confirm), I'd revert this